### DISCUSSION TITLE
編集画面でスクロールが必要な場合、ツールバーが隠れるようになる

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#423] [要望・アイデア・不具合]
編集画面でスクロールが必要になると、以下の画像のようにツールバー (?) が隠れるようになります。

![image](https://user-images.githubusercontent.com/75878492/214872603-ee804ec1-4c7f-4519-b6ed-a028b4fb7a57.png)

## Version

- Google Chrome: 109.0.5414.120（Official Build） （64 ビット）
- Windows 11

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
ご報告ありがとうございます！
こちら修正中ですので、もうしばらくお待ちいただけますと幸いです🙇

**Reply by tanaka-asami415** [reply]
@Yarakashi-Kikohshi 
修正が完了しました！🙇

**Reply by yarakos95** [reply]
対応ありがとうございます！
問題が修正されていることを確認しました。

どうやら私が提起する前から認知していた問題だったようで、せかしてしまうようなことをしてしまい申し訳ありませんでした。

**Reply by tanaka-asami415** [reply]
@Yarakashi-Kikohshi 
とんでもないです！ご確認いただきありがとうございます🙇
